src/App.js

```diff
@@ -16,6 +17,7 @@ function App() {
         <Route path="/" element={<Articles />} />
         <Route path={`/topics/:topic_name`} element={<Topics />} />
         <Route path={`/articles/:article_id`} element={<SingleArticle />} />
+        <Route path={`/articles/:article_id/addcomment`} element={<AddNewComment />} />
```

Comment:
Do you need this path now that your add comment component is on the same page?

Reply:
done! Completely forgot about it!